services/transactionpool/committed_pool.go

```diff
@@ -11,12 +12,27 @@ import (
 type committedTxPool struct {
 	transactions map[string]*committedTransaction
 	lock         *sync.RWMutex
+
+	metrics *committedPoolMetrics
+}
+
+type committedPoolMetrics struct {
+	transactionCountGauge *metric.Gauge
```

Comment:
why do you call this field `fooGauge`? we know it's a `Gauge`

Reply:
Fixed.